### ISSUE TITLE
Redirects to do during PI 14.2

### ISSUE THREAD

**Issue by kathycarothers** [fecgov/fec-cms#4471]
### Summary

**What we're after:**
We are getting off of transition. Old files need to be redirected to their new homes before we can take 'em down.

### Related issues

_List any relevant related issue(s)_

- #4381 Hold a meeting to get off transition 

### Completion criteria

_List any considerations the tech team should know. Additionally, any specific tech steps can be included here._

- [x] Redirects are merged into the release (See spreadsheet for redirects) 
- [x] 2021 spreadsheet is updated
- [x] Testing ticket is created to be done in the next sprint after deploy of releases. Also will need to check www redirect after release deploy.

### Future work

Once redirects have been deployed need to alert frontend to delete these transition files in the folder. See ticket #4380

**Comment by kathycarothers** [comment]
I noticed the redirects for the missing agendas (2010, 2012 and 2016) were already inputted.
